### ISSUE TITLE
Code Review

### ISSUE THREAD

**Issue by MrMace** [jaronmanyama/blackjackbuddy#18]
Code Review Done By: Matt Mace.
Reviewing Jaron Manyama project.
Code Review Black Jack Buddy

Design - Looking at the code it is well designed. It is easily readable. It is one of the few times I have looked at code and understood fully what each thing was doing.

Functionality - If someone needed to take the code and use it on their own projects they  would have an easy time finding what they need without the hassle of trying to decipher it.

Complexity - The code is not anymore complex than it needs to be.

Testing - Unit testing is shown to be used within the project adding via a separate file.

Naming - Classes, functions , and variables all have good naming conventions that allowed me to pinpoint what they were for and did. 

Comments - Jaron did an excellent job of providing comments for everything in his program. Which even without I would've known what each line was doing, but this just reinforced it. 

Styles - Jaron did an awesome job of styling the code layout to be up to what python style guide recommends. If I had to nitpick I would say there are some points that could improve here but are miniscule. Example:
  # Aligned with opening delimiter.
foo = long_function_name(var_one, var_two,
                         	        var_three, var_four)

Doing this for the extra long arrays, I wouldn’t have to scroll to see them all.  Again that is just nitpicking.

Consistency - The code stays consistent the whole file.

Documentation - Jaron has all the documentation needed and it is legible. It provided me with all the information I needed, and answered any questions that I had thought to myself. 

Summary
In summary Jarons code was clean and percises it was easy to read and understand. I tried my best after spending  hours  reviewing, to find something. I was however unsuccessful in my attempt because his code was very professionally put together.  There is nothing he did that I would have done differently.
